doozerlib/distgit.py

```diff
@@ -1797,9 +1797,14 @@ def _get_csv_file_and_refs(self, csv_config):
         subdir = csv_config.get('bundle-dir', f'{gvars["MAJOR"]}.{gvars["MINOR"]}')
         manifests = os.path.join(self.distgit_dir, csv_config['manifests-dir'], subdir)
 
-        refs = os.path.join(manifests, 'image-references')
-        if not os.path.isfile(refs):
-            raise DoozerFatalError('{}: file does not exist: {}'.format(self.metadata.distgit_key, refs))
+        refs = None
+        ref_candidates = [os.path.join(path, 'image-references') for path in [csv_config['manifests-dir'], subdir, manifests]]
```

Comment:
these would need to be updated to absolute paths. I cleaned it up a bit at https://github.com/thegreyd/doozer/commit/60ffb2d9cbc284d3e88d1d0415b2ddbd8847c46b

Reply:
Also are we changing the behavior for finding csvs as well?

Reply:
nope. just the single file.

Reply:
cp'ed that commit